webdriver-spec.html

```diff
@@ -6443,14 +6443,14 @@
   <td>"<code>expiry</code>"
   <td>"<code>Max-Age</code>"
   <td>✓
-  <td>When the cookie expires,
-   specified in seconds since <a>Unix Epoch</a>.
-   Defaults to 20 years into the future
-   if omitted when <a>adding a cookie</a>.
+  <td>When the cookie expires, specified in seconds since
+   <a>Unix Epoch</a>.  Must not be set if omitted when
+   <a>adding a cookie</a>.
```

Comment:
I think this is incompatible with the description in the ‘table for cookie conversion’, which claims cookie expiry is non-optional.

The problem here is that we need to describe how to _deserialise_ a JSON Object we get from the client to a cookie representation as well as how to _serialise_ one from the browser’s cookie store. These two have different expectations with regards to required fields, as I understand it?